### PR TITLE
Spearhead: refine typography styles

### PR DESCRIPTION
**Pull request by jffng**
This PR addresses #2592.

### CODE REVIEW

**Comment by MaggieCabrera** [comment]
On the sticky post not all html elements are white. The `ul` tag and the heading are still gray (the inherit the color from the body)

Screenshot 2020-10-15 at 09 15 18

**Comment by MaggieCabrera** [comment]
The navigation links font size is still 16px for me, not sure if I'm missing something:

Screenshot 2020-10-15 at 09 17 59

**Comment by jffng** [comment]
Good catches, thanks @MaggieCabrera!

File: spearhead/variables.css [hunk 1]
```diff
@@ -25,7 +25,7 @@
 	--global--content-width: 900px;
 	/* Line Height */
 	--global--line-height-base: 1;
-	--global--line-height-body: 1.7;
+	--global--line-height-body: 1.6;
```
Comment (kjellr) [inline spearhead/variables.css:28]: Sorry — I must've mistyped in the original issue. This should be `1.8`, not `1.6`.

**Review by kjellr — COMMENTED** [review]
This looks good! Left one small comment above. Besides that, the only additional note I have is that we should increase the font size of the post date on single post pages to match `--global--font-size-base`: 


Current:
Screen Shot 2020-10-15 at 1 23 54 PM

Recommended: 
Screen Shot 2020-10-15 at 1 26 23 PM

(If at all possible, it would be great to change the date format so that the month isn't capitalized too — `Jun` instead of `JUN` — but that can happen separately.)

**Pull request merged**